docs/index.md

```diff
@@ -19,22 +19,27 @@ ONNX Runtime is an accelerator for machine learning models with multi platform s
 ## Contribute and Customize
 
 |:----------------------------------------------------------------------------------|:-----------------------------------------------------------------------------------|
-|  <span class="fs-5"> [Build ORT Packages](./build){: .btn  .mr-4 target="_blank"} </span>         | <span class="fs-5">[ONNX Runtime GitHub](https://github.com/microsoft/onnxruntime){: .btn target="_blank"} </span>  |
+|  <span class="fs-5"> [Build ORT Packages](./build){: .btn  .mr-4 target="_blank"} </span>| <span class="fs-5">[ONNX Runtime GitHub](https://github.com/microsoft/onnxruntime){: .btn target="_blank"} </span>  | 
 
 
 ---
 
-
 ## QuickStart Template
 
 |:----------------------------------------------------------------------------------|:-----------------------------------------------------------------------------------|
-|  <span class="fs-5"> [ORT Web JavaScript Site Template using NextJS](https://github.com/microsoft/onnxruntime-nextjs-template){: .btn  .mr-4 target="_blank"} </span>         | 
+|  <span class="fs-5"> [ORT Web JavaScript Site Template](https://github.com/microsoft/onnxruntime-nextjs-template){: .btn  .mr-4 target="_blank"} </span>         |  <span class="fs-5"> [ORT C# Console App Template](https://github.com/microsoft/onnxruntime-csharp-cv-template){: .btn  .mr-4 target="_blank"} </span>         | 
 
 
 ---
 
+## ONNX Runtime YouTube Channel
```

Comment:
This might get a bit lost on this page. Wondering if we should add to the links across the top?

Reply:
I was wondering if that was a good spot or not. So you think putting the links above the video would be better?

Reply:
I also considered just adding it to the "How to use ONNX Runtime (ORT)" table instead of it being its own section. Thoughts?